vm-mock/vm.js

```diff
@@ -364,6 +364,14 @@ export default function createMockedABI(
 
         return newArrayBuffer(keys);
       },
+
+      assembly_script_get_owned_addresses() {
+        return newString(`[ ${callerAddress} , ${contractAddress} ]`);
```

Comment:
What is ownedAddresses supposed to return ?

Reply:
It's not documented yet, I just deduced that it was an address array in json format: 
/**
 * Returns owned addresses.
 *
 * TODO:
 * - explain function purpose
 * - explain format
 *
 */
 
 So for now I return an array of address in json format. I plan to improve the comment in the documentation issue